### PR TITLE
fastdds Python tool (#1252) > 2.0.x [9025]

### PR DESCRIPTION
**Pull request by LuisGP**
Backport to 2.0.x

### CODE REVIEW

File: test/CMakeLists.txt [hunk 1]
```diff
@@ -58,3 +58,10 @@ endif()
 if(PERFORMANCE_TESTS)
     add_subdirectory(performance)
 endif()
+
+###############################################################################
+# System tests
+###############################################################################
+if(PERFORMANCE_TESTS)
```
Comment (MiguelCompany) [inline test/CMakeLists.txt:65]: It seems we missed something here on the revision of the original PR. I think this should be a different option `SYSTEM_TESTS`

Comment (LuisGP) [reply test/CMakeLists.txt:65]:
```suggestion
if(SYSTEM_TESTS)
```

Comment (MiguelCompany) [reply test/CMakeLists.txt:65]: And also adding the option on line 19

File: test/CMakeLists.txt [hunk 2]
```diff
@@ -17,6 +17,7 @@ check_gtest()
 check_gmock()
 
 option(PERFORMANCE_TESTS "Activate the building and execution of performance tests" OFF)
+option(SYSTEM_TESTS "Activate the building and execution of system tests" OFF)
```
Comment (MiguelCompany) [inline test/CMakeLists.txt:20]:
```suggestion
option(SYSTEM_TESTS "Activate the building and execution of system tests" ON)
```

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/2461/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/2264/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/2452/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Linux/2540/)
* Mac [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Mac/2351/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/FastRTPS%20Manual%20Windows/2547/)

**Review by MiguelCompany — APPROVED** [review]

**Pull request merged**